libs/dal/src/repositories/notification-template/notification-template.repository.ts

```diff
@@ -29,6 +29,7 @@ export class NotificationTemplateRepository extends BaseRepository<
       _environmentId: environmentId,
       _organizationId: organizationId,
       origin: { $in: [ResourceOriginEnum.NOVU_CLOUD] },
+      type: { $in: [ResourceTypeEnum.BRIDGE] },
```

Comment:
I'm not entirely sure, but I think doing origin: ResourceOriginEnum.NOVU_CLOUD and type: ResourceTypeEnum.BRIDGE without the $in might be slightly more optimized